### ISSUE TITLE
Docs: some internal urls contain redundant dash and number

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#9892]
### Description

On the page https://dev.handsontable.com/docs/next/react-data-grid/cell-renderer/#cell-renderer
some URLs for headers contains redundant dash and number: "-X":
https://dev.handsontable.com/docs/next/react-data-grid/cell-renderer/#overview-2
https://dev.handsontable.com/docs/next/react-data-grid/cell-renderer/#using-a-cell-renderer-2
https://dev.handsontable.com/docs/next/react-data-grid/cell-renderer/#overview-3
https://dev.handsontable.com/docs/next/react-data-grid/cell-renderer/#using-a-cell-renderer-2
https://dev.handsontable.com/docs/next/react-data-grid/cell-renderer/#rendering-custom-html-in-header-2
https://dev.handsontable.com/docs/next/react-data-grid/cell-renderer/#adding-event-listeners-in-cell-renderer-function-2
https://dev.handsontable.com/docs/next/react-data-grid/cell-renderer/#related-api-reference-2
### Version
5b6a08d5f5a78507e943c5ceda2a00d407681578 
Staging environment

**Comment by warpech** [comment]
Thank you for noticing. These numbers are the result of conditioning - the MD file contains headings with the same text, some of which are only for JS, while the other are only for React. The pool for these ids is common.

However, I think that it was discussed in issues https://github.com/handsontable/handsontable/issues/9750 and https://github.com/handsontable/handsontable/issues/9759 and fixed by PR https://github.com/handsontable/handsontable/pull/9783.

@jansiegel can you refresh our memory if this is fixable? Anyway, I don't think it is a big problem, as long as TOC works.

**Comment by aninde** [comment]
I don't think this is a big problem, just a lack of consistency with other sites. These additional numbers also appear in other headers on other sites, but they are hard to find without automation.

**Comment by jansiegel** [comment]
@warpech, @aninde Should be fixed in #9898.
